tests/metrics/estimate_gas.py

```diff
@@ -294,8 +294,8 @@ def transact(function, transaction):
     #
     # Create policy
     #
-    policy_id_1 = os.urandom(int(Policy.POLICY_ID_LENGTH))
-    policy_id_2 = os.urandom(int(Policy.POLICY_ID_LENGTH))
+    policy_id_1 = os.urandom(int(Policy.ID_LENGTH))
```

Comment:
Re: My question about the separate constant.

I guess this might be the reason for the separate constant i.e. to not import something from the blockchain module...?